user-journey-tests/specs/destination/destinationAnswers.spec.js

```diff
@@ -0,0 +1,85 @@
+import {
+  loadPageAndVerifyTitle,
+  selectElement,
+  validateElementVisibleAndText,
+  waitForPagePath
+} from '../../helpers/page.js'
+import completeDestinationTest from '../../helpers/testHelpers/destination.js'
+import completeOriginTaskAnswers from '../../helpers/testHelpers/movementLicence.js'
+import destinationAnswersPage from '../../page-objects/destination/destinationAnswersPage.js'
+import destinationSelectionPage from '../../page-objects/destination/destinationSelectionPage.js'
+import generalLicencePage from '../../page-objects/destination/generalLicencePage.js'
+import landingPage from '../../page-objects/landingPage.js'
+import taskListPage from '../../page-objects/taskListPage.js'
+
+describe('Check your answers test - destination', () => {
+  // eslint-disable-next-line no-undef
+  beforeEach('Navigate to check answers page', async () => {
+    await browser.reloadSession()
+    await completeOriginTaskAnswers()
+    await loadPageAndVerifyTitle('', landingPage.pageTitle)
+  })
+
+  it('Should verify slaughter answer and change link', async () => {
+    await completeDestinationTest('slaughter')
+    await validateElementVisibleAndText(
+      destinationAnswersPage.destiationValue,
+      'Slaughter'
+    )
+
+    await selectElement(destinationAnswersPage.changeDestinationLink)
+    await expect(destinationSelectionPage.slaughterRadio).toBeSelected()
+
+    await destinationSelectionPage.selectContinue()
+    await selectElement(generalLicencePage.continueLink)
```

Comment:
no test to check use genral licence page looks correct?

Reply:
There's a test already for that